src/test/counter.test.js

```diff
@@ -0,0 +1,17 @@
+const counter = require('../counter.js');
+
+describe('Counter Function Test', () => {
+  test('Return testArray length', () => {
+    const testArray = [1, 3, 8, 0, 10];
+    const testLength = counter(testArray);
+    expect(testLength).toBe(5);
+  });
+});
+
+describe('Counter Function Test', () => {
+  test('Return testArray length', () => {
+    const testArray = [{ name: 'Rida' }, { lastName: 'Arif' }, { DOB: '07-06-98' }];
+    const testLength = counter(testArray);
+    expect(testLength).toBe(3);
```

Comment:
It's better to use one describe because you are testing the same function. You can copy the second test and paste it in the first describe and remove the second.